pkg/cloudinit/controlplane_init.go

```diff
@@ -27,7 +27,7 @@ const (
 {{template "files" .WriteFiles}}
 runcmd:
 {{- template "commands" .PreK3sCommands }}
-  - 'curl -sfL https://get.k3s.io | INSTALL_K3S_VERSION=%s sh -s - server'
+  - 'curl -sfL https://get.k3s.io | INSTALL_K3S_VERSION=%s sh -s - server && mkdir -p /run/cluster-api && echo success > /run/cluster-api/bootstrap-success.complete'
```

Comment:
what is your opinion of just adding this as a separate command rather than chaining it?

I prefer this approach as i find it more readable and easier to debug on failed nodes, but that is just my two cents.

Reply:
A separate command would be incorrect, please check the comment above.